src/queries/get/InitQueries.ts

```diff
@@ -346,8 +335,10 @@ export async function getLinksConfig(
       "?diagram og:id ?diagramID.",
       "?diagram og:representation ?representation.",
       "}",
-      `${qb.i(AppSettings.contextIRI)} ?linkPredicate ?graph.`,
-      `values ?linkPredicate { <${linkPredicates.join("> <")}> }`,
+      `${qb.i(AppSettings.contextIRI)} <${parsePrefix(
+        "d-sgov-pracovní-prostor-pojem",
+        "odkazuje-na-přílohový-kontext"
```

Comment:
This searches for _workspace_ diagram contexts of _workspace_ vocabulary contexts.